apps/nextjs-website/src/components/organisms/WebinarQuestionsTemplate/WebinarQuestionsTemplate.tsx

```diff
@@ -69,24 +72,28 @@ const WebinarQuestionsTemplate = ({
                 </TableRow>
               </TableHead>
               <TableBody>
-                {sortedQuestions.map((row) => (
+                {sortedQuestions.map((webinarQuestion) => (
                   <WebinarQuestionRow
-                    key={row.createdAt.toJSON()}
-                    question={row}
+                    key={webinarQuestion.createdAt.toISOString()}
+                    question={webinarQuestion}
                     userName={userName}
                     onHide={async (hide) =>
-                      await updateQuestion(
-                        row,
-                        'hide',
-                        hide ? userName : undefined
-                      )
+                      await updateWebinarQuestion({
+                        webinarId: webinarQuestion.webinarId,
+                        createdAt: webinarQuestion.createdAt,
+                        hiddenBy: hide
```

Comment:
I would change the name of the param, hiddenBy to me is the user making the operation, not the operation itself, I'd use something like webinarQuestionUpdate, with an additional param to trak if its hidden or highlighted

Reply:
Can you provide an example to help me understand your suggestion?

Reply:
Sure! :)
I think that having a name `hiddenBy` and then an object like `{ operation: 'update', value: userName }` it's a little bit confusing. With a name like that I am expecting to find something related to the user who made the operation and not not the operation itself. 

So i thought we could have it named differently... something like `hiddenUpdateOperation`, or `higlighedUpdateOperation`

Reply:
Oki. I revised the `WebinarQuestionUpdate` and `WebinarQuestion` types to clarify the usage of the `updateWebinarQuestion` function. Let me know if this clarification helps